src/main/java/it/gov/pagopa/payments/service/PartnerService.java

```diff
@@ -206,15 +213,21 @@ public PaSendRTRes paSendRT(PaSendRTReq request) {
 
   @Transactional
   public PaSendRTV2Response paSendRTV2(PaSendRTV2Request request) {
+    try {
+      PaymentOptionModelResponse paymentOption = managePaSendRtRequest(request);
 
-    PaymentOptionModelResponse paymentOption = managePaSendRtRequest(request);
-
-    if (!PaymentOptionStatus.PO_PAID.equals(paymentOption.getStatus())) {
-      log.error(
-          "[paSendRTV2] Payment Option [statusError: {}] [noticeNumber={}]",
-          paymentOption.getStatus(),
-          request.getReceipt().getNoticeNumber());
-      throw new PartnerValidationException(PaaErrorEnum.PAA_SEMANTICA);
+      if (!PaymentOptionStatus.PO_PAID.equals(paymentOption.getStatus())) {
+          log.error(
+                  "[paSendRTV2] Payment Option [statusError: {}] [noticeNumber={}]",
+                  paymentOption.getStatus(),
+                  request.getReceipt().getNoticeNumber());
+          throw new PartnerValidationException(PaaErrorEnum.PAA_SEMANTICA);
+      }
+    } catch (PartnerValidationException e) {
+      // Re-throw the exception unless it's a suppressed fault code error from station included in the suppression list.
+      if (!suppressedErrorsValues.contains(e.getError().getFaultCode()) || !stationsWithSuppressedErrors.contains(request.getIdStation())) {
+        throw e;
+      }
```

Comment:
Just a micro-optimization, I would set `!stationsWithSuppressedErrors.contains(request.getIdStation()` as first check to do in this `if`, in order to avoid having to perform a double check on “non-special” stations almost every time.  
Also, what do you think about adding a log (even at `DEBUG` level) to describe the fact that the error has been suppressed?